car_rental/api/models.py

```diff
@@ -1,4 +1,8 @@
+from email.policy import default
+from pyexpat import model
+from random import choices
```

Comment:
Is this needed?

Reply:
no I don't think they are...I'm not sure what the reason for those imports for VS Code were.